index.bs

```diff
@@ -413,9 +413,12 @@ The user agent may delay the reporting of layout shifts after a
 until such time as it is known that the event does not begin a fling or scroll
 gesture.
 
-The <a href="https://www.w3.org/TR/uievents/#event-type-mousemove">mousemove</a> and
-<a href="https://www.w3.org/TR/pointerevents/#the-pointermove-event">pointermove</a>
-events are also not excluding inputs.
+The <a href="https://www.w3.org/TR/uievents/#event-type-mousemove">mousemove</a> events after the
+<a href="https://www.w3.org/TR/uievents/#event-type-mousedown">mousedown</a> event and the
+<a href="https://www.w3.org/TR/pointerevents/#the-pointermove-event">pointermove</a> events after
+the <a href="https://www.w3.org/TR/pointerevents/#the-pointerdown-event">pointerdown</a> event,
+which trigger dragging elements around or resizing elements but not scrolling, should be excluding
+inputs.
```

Comment:
Should this also mention that anything with pointercancel would not be an excluding input? Just move after down is not sufficient, as that would include scrolls and fling/flick.

Reply:
Yes, it is clearer to add that  pointercancel means scrolling happen, and it is not an excluding input. Thank you.